src/sdk/types.ts

```diff
@@ -212,7 +220,17 @@ export type Category =
 
 export type PlatformDisplayConfig = {
   name: string;
-  iconURI: string;
+  /** @deprecated */
+  iconURI?: string;
+  icons: {
+    defaultURI: string;
+    fullURI?: string;
+    boxURI?: string;
```

Comment:
How is each of them being used?

Reply:
Let me add some comments - but the box logo is the 'condensed' logo that we use on the integrations page. The fullURI is the expanded version that usually includes the name of the company in it that we use in the modal and on the unified API pages on the website.

Down for whatever naming these are just the first things I thought of.